monetio/obs/ish_lite.py

```diff
@@ -4,7 +4,15 @@
 
 
 def add_data(
-    dates, box=None, country=None, state=None, site=None, resample=True, window="H", n_procs=1
+    dates,
+    box=None,
+    country=None,
+    state=None,
+    site=None,
+    resample=True,
+    window="H",
```

Comment:
I fixed it so the resample works now, but since it is already hourly, shouldn't it be false by default?

Reply:
Good idea